R/by_aesthetics.R

```diff
@@ -247,3 +247,47 @@ by_lwd = function(ngrps, type, lwd=NULL) {
   
   return(lwd)
 }
+
+
+by_bg = function(
+  adjustcolor,
+  alpha,
+  bg,
+  by,
+  by_continuous,
+  by_ordered,
+  col,
+  fill,
+  ngrps,
+  palette,
+  ribbon.alpha,
+  type) {
+
+  if (is.null(bg) && !is.null(fill)) bg = fill
```

Comment:
Should this go higher up in the main tinyplot function? That way, we can also drop the extra fill argument for this `by_bg` function (as well as the input on l. 665 below).